src/Mod/TechDraw/Gui/DlgPrefsTechDrawAdvanced.ui

```diff
@@ -599,8 +599,6 @@ Each unit is approx. 0.1 mm wide</string>
    <header>Gui/PrefWidgets.h</header>
   </customwidget>
  </customwidgets>
- <resources>
-  <include location="Resources/TechDraw.qrc"/>
```

Comment:
Why this deletion?

By the way, due to Werner's recent changes for PrefQuantitySpinBox one maybe needs to recompute the Qt designer plugin. TD does not yet use use PrefQuantitySpinBox but in future you might want to use it.